sleepsta/src/components/Profile/Profile.js

```diff
@@ -18,7 +18,6 @@ import Notifications, { notify } from "../Notifications/index";
 import styled from "styled-components";
 
 const DarkCard = styled(Card)`
-  background-color: #0a1429;
```

Comment:
could we keep the background color on the dark card on profile

Reply:
I had fixed the issue along with new changes